src/chainstate/stacks/boot/costs-2.clar

```diff
@@ -226,6 +226,21 @@
 (define-read-only (cost_xor (n uint))
     (runtime u170))
 
+(define-read-only (cost_bitwise_and (n uint))
```

Comment:
Please remove all changes to `costs-2.clar`.  It's consensus-critical data that cannot be changed.

Reply:
Done